lib/presentation/widgets/m3_widgets/custom_alert_dialog.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:trip_tally/presentation/theme/app_colors.dart';
+import 'package:trip_tally/presentation/utils/enums/context_extensions.dart';
+
+class CustomAlertDialog extends StatelessWidget {
+  const CustomAlertDialog({
+    required this.title,
+    required this.info,
+    required this.isConfirmation,
```

Comment:
you can add a default value to true, because most of the times it will confirmation